tests/burn.idempotency.test.ts

```
@@ -0,0 +1,118 @@
+    const res: any = {
+      status: jest.fn().mockReturnThis(),
+      json: jest.fn().mockReturnThis(),
+    };
+    const next = jest.fn();
+
+    await burnAcbu(req, res, next);
+
+    expect(prismaMock.transaction.create).not.toHaveBeenCalled();
+    expect(res.status).toHaveBeenCalledWith(200);
+    expect(res.json).toHaveBeenCalledWith(
+      expect.objectContaining({ transaction_id: "tx-1", blockchain_tx_hash: blockchainTxHash }),
```

ESLint

[error] Replace `·transaction_id:·"tx-1",·blockchain_tx_hash:·blockchainTxHash` with `⏎········transaction_id:·"tx-1",⏎········blockchain_tx_hash:·blockchainTxHash,⏎·····`

(prettier/prettier)